components/notion/sources/common/common.mjs

```diff
@@ -0,0 +1,23 @@
+import notion from "../../notion.app.mjs";
+import constants from "../common/constants.mjs";
+
+export default {
```

Comment:
When a [`common`](https://pipedream.com/docs/components/guidelines/#common-files-optional) module is placed in the common folder (`sources/common/`) rather than `sources/`, I think the convention for this repo is to name it _`base.mjs`_ if it's inherited by the other `common` modules/all sources (e.g. [sengrid](https://github.com/PipedreamHQ/pipedream/tree/master/components/sendgrid/sources/common)), or a more use-specific name like [`webhook.js`](https://github.com/PipedreamHQ/pipedream/blob/55236b3aa993cbcb545e245803d8654c6358b0a2/components/eventbrite/sources/common/webhook), _`polling.mjs`_, or [`tweets.mjs`](https://github.com/PipedreamHQ/pipedream/blob/55236b3aa993cbcb545e245803d8654c6358b0a2/components/twitter/sources/common/tweets.mjs).

Reply:
Addressed